src/richie-front/js/data/configureStore.ts

```diff
@@ -1,16 +1,25 @@
-import { applyMiddleware, createStore } from 'redux';
+import { applyMiddleware, compose, createStore } from 'redux';
 import createSagaMiddleware from 'redux-saga';
 
+import { Config } from '../types/config';
 import { rootReducer, RootState } from './rootReducer';
 import { rootSaga } from './rootSaga';
 
 const sagaMiddleware = createSagaMiddleware();
 
+declare const CONFIG: Config;
```

Comment:
I think this would be better as an import. We could have a file that does EG:

```
declare const CONFIG: Config;
export CONFIG = CONFIG;
```

Also maybe we can keep the name settings as in the Django part of the project? What do you think?

Reply:
> Also maybe we can keep the name settings as in the Django part of the project? 

like this `ENABLE_DEV_TOOLS` ?

For the declare part, I created a `@types` folder is src and I use it in `tsconfig.json`

Reply:
I meant naming the variable `settings` instead of `CONFIG` but I see this name is forced by the webpack plugin. Ignore this comment :)

Reply:
No it's possible to change it https://www.npmjs.com/package/config-webpack#configuring

Reply:
Now that you say it, existing settings are formatted like Django in all-caps snake-case. I think we can keep it that way.